tests/test_ffmpeg.py

```diff
@@ -224,6 +226,44 @@ def test_write_not_contiguous(test_images):
             assert diff.mean() < 2.5
 
 
+def has_audio_streams(file_path):
+    command = ["ffprobe", "-show_streams", "-print_format", "json", file_path]
+    output = check_output(command)
+    parsed = json.loads(output)
+    streams = parsed["streams"]
+    audio_streams = list(filter((lambda x: x["codec_type"] == "audio"), streams))
+    return len(audio_streams) > 0
+
+
+def test_write_audio(test_images, tmp_path):
+    in_filename = test_images / "realshort.mp4"
+    out_filename = tmp_path / "realshort.mp4"
+    out_filename_audio = tmp_path / "realshort_audio.pm4"
+
+    in_file = []
+    with iio.read(in_filename, "ffmpeg") as R:
+        for i in range(10):
+            im = R.get_next_data()
+            in_file.append(im)
+
+    # Write without audio
+    with iio.save(out_filename, "ffmpeg") as W:
+        for im in in_file:
+            W.append_data(im)
+
+    # Now write with audio to preserve the audio track
+    with iio.save(
+        out_filename_audio, format="ffmpeg", audio_path=in_filename.as_posix()
+    ) as W:
```

Comment:
I must admit that I am not an expert on imageio-ffmpeg internals, so I may be misunderstanding things.

If I call this code as is, will it copy (transmux) the full audio from the input file into the output file, or will it truncate it to be in sync with the video frames, i.e., since we are only writing a fraction of a second of video, will we only copy a fraction of the audio or will we get a full copy of the audio track?

Reply:
That's a very good question. Will actually try it out and back to you on that.

Reply:
Ah shoot, I just pushed with the adapted test and realized I hadn't check this. And yes, we'll get the full audio track while the video stops at the last frame. Will try to think of a way to circumvent this.

Reply:
I agree that `imageio-ffmpeg` would be the place where this has to be implemented. The reason for this is that we will likely have to truncate the last audio package to match the exact time at which the last frame is sent.

That said, not having this feature is no dealbreaker as long as we clearly document this limitation. Could you write a sentence or two about this in the `Notes` section of the docs? Something along the lines of: "This will copy the full audio stream, so make sure your video length matches that."

Reply:
Will do